### ISSUE TITLE
nth-child in IE

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#202]
```
What steps will reproduce the problem?

1. Create a table with thead and tbody sections (index.html);
2. Define nth-child odd and even background colors for rows inside 
tbody(index.css);

What is the expected output? What do you see instead?

The odd rows, inside tbody's section, should be darker. It seems that in IE 
it starts counting from the first table row instead of the first row inside 
tbody's section. In Chrome, FireFox and Safari works as expected.

What version of the product are you using? On what operating system?

Windows XP SP3. Browsers:

Chrome v3.0.195.25;
FireFox v3.5.3;
Internet Explorer vi6.0.2900.5512.xpsp_sp3_gdr.090206-1234;
Safari v3.2.2(525.28.1);

Please provide any additional information below.

```

Original issue reported on code.google.com by `andre.r....@gmail.com` on 6 Oct 2009 at 2:36

Attachments:
- [chrome_3_0_195_25.png](https://storage.googleapis.com/google-code-attachments/ie7-js/issue-202/comment-0/chrome_3_0_195_25.png)
- [ff_3_5_3.png](https://storage.googleapis.com/google-code-attachments/ie7-js/issue-202/comment-0/ff_3_5_3.png)
- [ie_6_0_2900_5512_xpsp_sp3_gdr_090206-1234.png](https://storage.googleapis.com/google-code-attachments/ie7-js/issue-202/comment-0/ie_6_0_2900_5512_xpsp_sp3_gdr_090206-1234.png)
- [safari_3_2_2(525_28_1).png](https://storage.googleapis.com/google-code-attachments/ie7-js/issue-202/comment-0/safari_3_2_2%28525_28_1%29.png)
- [index.html](https://storage.googleapis.com/google-code-attachments/ie7-js/issue-202/comment-0/index.html)
- [index.css](https://storage.googleapis.com/google-code-attachments/ie7-js/issue-202/comment-0/index.css)

**Comment by GoogleCodeExporter** [comment]
```
This fixed in version 2.1.
```

Original comment by `dean.edw...@gmail.com` on 11 Feb 2010 at 6:13
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Nice work.
```

Original comment by `andre.r....@gmail.com` on 11 Feb 2010 at 6:20
